packages/app/src/app/hooks/useSubscription.ts

```diff
@@ -82,6 +82,7 @@ export const useSubscription = () => {
   const maxPublicProjects = activeTeamInfo?.limits?.maxPublicProjects;
 
   const hasMaxPublicRepositories =
+    isTeamSpace &&
```

Comment:
isn't the same restriction valid for personal spaces?

Reply:
@tristandubbeld @alexnm on the pricing page, we mention unlimited public repositories for personal free accounts.